MICADO/docs/example_notebooks/MICADO_4_stars.ipynb

```diff
@@ -0,0 +1,256 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "c91d6ba9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.colors import LogNorm\n",
+    "from astropy import units as u\n",
+    "\n",
+    "import scopesim_templates as sim_tp\n",
+    "import scopesim as sim"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "04224d82",
+   "metadata": {},
+   "source": [
+    "# Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "e962001c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Gijs's local computer\n",
+    "# sim.download_packages([\"Armazones\", \"ELT\", \"MAORY\", \"MICADO\"])\n",
+    "\n",
+    "# Kieran's computer -> redirects ScopeSim to folder containing IRDB \n",
+    "# MicadoWise could do this too and clone daily from github to save running time\n",
+    "sim.rc.__config__[\"!SIM.file.local_packages_path\"] = \"F:\\Work\\irdb\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eaf2e74a",
+   "metadata": {},
+   "source": [
+    "# Make the MICADO optical train"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "418bfe52",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cmds = sim.UserCommands(use_instrument=\"MICADO\", set_modes=[\"SCAO\", \"IMG_1.5mas\"])\n",
+    "\n",
+    "# EXPTIME = 3600 = ndit * dit\n",
+    "cmds[\"!DET.dit\"] = 30\n",
+    "cmds[\"!DET.ndit\"] = 120\n",
```

Comment:
For a proper observation with correct headers, it is required to put some more information as input to ScopeSIM, see the other notebooks as examples.